packages/datadog-plugin-cypress/src/plugin.js

```diff
@@ -212,6 +218,16 @@ module.exports = (on, config) => {
       testSpanMetadata[TEST_CODE_OWNERS] = codeOwners
     }
 
+    if (isUnskippable) {
+      hasForcedToRunTests = true
+      testSpanMetadata[TEST_ITR_UNSKIPPABLE] = 'true'
+    }
+
+    if (isForcedToRun) {
+      hasUnskippableTests = true
+      testSpanMetadata[TEST_ITR_FORCED_RUN] = 'true'
+    }
```

Comment:
If `isForcedToRun` is `true`, then `hasUnskippableTests` should be implied `true` and shouldn't need to be set in the second `if` statement, no?

Reply:
yeah this is wrong. Good catch!

Reply:
fixed and added a test in https://github.com/DataDog/dd-trace-js/pull/3684/commits/40ce56fe99741d7538102b630d247cf95f6ba293